Student_BCOs/(FinalUpdate)Juntian_Wei_BCO_000549.json

```diff
@@ -0,0 +1,434 @@
+                "name": "R",
+                "version": "3.0.1 or higher",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://mirror.fcaglp.unlp.edu.ar/CRAN/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "gcc",
+                "version": "9.2",
+                "uri": {
+                    "access_time": "",
+                    "uri": "https://gcc.gnu.org/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "make",
+                "version": "4.2",
+                "uri": {
+                    "access_time": "",
+                    "uri": "http://ftp.gnu.org/gnu/make/",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            },
+            {
+                "name": "g++",
+                "version": "2.69",
+                "uri": {
+                    "access_time": "",
+                    "uri": "https://launchpad.net/~ubuntu-toolchain-r/+archive/ubuntu/test",
+                    "sha1_checksum": "000528checksum",
+                    "filename": ""
+                }
+            }
+        ],
+        "script": []
+    },
+    "parametric_domain": [
+        {
+            "step": "",
+            "param": "",
+            "value": ""
+        }
+    ],
+    "io_domain": {
+        "input_subdomain": [
+            {
+                "uri": {
+                    "access_time": "2013-12-13T00:00:00-0500",
+                    "uri": "www.example.com/iMirFiles/iMirInputFiles.FASTQ",
+                    "sha1_checksum": "000528checksum",
+                    "filename": "iMirInputFiles.fastq"
+                }
+            }
+        ],
+        "output_subdomain": [
+            {
+                "mediatype": "text/csv",
+                "uri": {
+                    "access_time": "2013-12-13T00:00:00-0500",
+                    "uri": "www.example.com/iMirFiles/SampleRNA_converted.rc",
+                    "sha1_checksum": "000528checksum",
+                    "filename": "SampleRNA_converted.rc"
+                }
+            },
+            {
+                "mediatype": "text/csv",
+                "uri": {
+                    "access_time": "2013-12-13T00:00:00-0500",
+                    "uri": "www.example.com/iMirFiles/trimmed_RNA.fa",
+                    "sha1_checksum": "000528checksum",
+                    "filename": "trimmed_RNA.fa"
+                }
+            },
+            {
+                "mediatype": "png",
+                "uri": {
+                    "access_time": "2013-12-13T00:00:00-0500",
+                    "uri": "www.example.com/iMirFiles/RNA_LengthDistribution.png",
+                    "sha1_checksum": "000528checksum",
```

Comment:
The ID for the submitted BCO is BCO_000549. 
Please modify this checksum accordingly.